xml/System.ComponentModel.DataAnnotations/DisplayFormatAttribute.xml

```diff
@@ -101,11 +107,19 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets or sets a value that indicates whether the formatting string that is specified by the <see cref="P:System.ComponentModel.DataAnnotations.DisplayFormatAttribute.DataFormatString" /> property is applied to the field value when the data field is in edit mode.</summary>
+        <summary>Gets or sets a value that indicates whether the formatting string that is specified by the           
+          <see cref="P:System.ComponentModel.DataAnnotations.DisplayFormatAttribute.DataFormatString" /> property is applied to the field value when the data field is in edit mode.
+        
+        </summary>
         <value>
-          <see langword="true" /> if the formatting string applies to the field value in edit mode; otherwise, <see langword="false" />. The default is <see langword="false" />.</value>
+          <see langword="true" /> if the formatting string applies to the field value in edit mode; otherwise,           
+          <see langword="false" />. The default is           
+          <see langword="false" />.
```

Comment:
Is there a reason to reformat this code like this? Similar cases are also below.

Reply:
I am guessing it is a vscode autoformat thing. I will fix them.